app/navigation/Navigator.scala

```diff
@@ -84,21 +84,30 @@ class Navigator @Inject() () {
     case ExplainHowYouWillUseMethodSixPage                => explainHowYouWillUseMethodSixPage
     case AdaptMethodPage                                  => adaptMethodPage
     case DeleteDraftPage                                  => _ => AccountHomeController.onPageLoad()
+    case WhoAreYouAgentPage                               => whoAreYouRouting
     case _                                                => _ => AccountHomeController.onPageLoad()
   }
 
   private def startApplicationRouting(userAnswers: UserAnswers): Call =
     userAnswers.get(AccountHomePage) match {
-      case Some(IndividualTrader)                    =>
-        RequiredInformationController.onPageLoad(userAnswers.draftId)
-      case Some(OrganisationAdmin)                   =>
-        RequiredInformationController.onPageLoad(userAnswers.draftId)
-      case Some(OrganisationAssistant) | Some(Agent) =>
-        WhatIsYourRoleAsImporterController.onPageLoad(NormalMode, userAnswers.draftId)
-      case _                                         =>
+      case Some(IndividualTrader)      =>
+        WhoAreYouAgentController.onPageLoad(NormalMode, userAnswers.draftId)
+      case Some(OrganisationAdmin)     =>
+        WhoAreYouAgentController.onPageLoad(NormalMode, userAnswers.draftId)
+      case Some(OrganisationAssistant) =>
+        WhoAreYouAgentController.onPageLoad(NormalMode, userAnswers.draftId)
+      case Some(Agent)                 =>
+        WhoAreYouAgentController.onPageLoad(NormalMode, userAnswers.draftId)
+      case _                           =>
         UnauthorisedController.onPageLoad
     }
```

Comment:
This was the only part with a conflict when rebasing. I took the original PRs changes and add the line for `Some(Agent)` as I believe that would've been the intention had the type existed

Reply:
I can see that this area needs a bit of a cleanup. I'm keen to merge this current PR as is, and raise a technical task to standardise how we display content when the same page looks different depending on the user journey (which may have happened already).